site/en/docs/extensions/mv3/service_workers/index.md

```diff
@@ -76,13 +75,15 @@ chrome.runtime.onInstalled.addListener(() => {
 
 ## Set up listeners {: #listeners }
 
-Structure background scripts around events the extension depends on. Defining functionally relevant
-events allows background scripts to lie dormant until those events are fired and prevents the
+Structure extension service workers around events the extension depends on. Defining functionally relevant
+events allows the service wroker to lie dormant until those events are fired and prevents the
 extension from missing important triggers.
 
-Listeners must be registered synchronously from the start of the page.
+Listeners must be registered at the start of your extension service worker. Notice that in this
+example the listener for creating bookmarks is at the top level of the script.
 
-```js
+{% Compare 'better' %}
+```js/8-10
```

Comment:
Looks like we might have an off by one error here; I'm assuming that the comment wasn't supposed to be included in the highlight range and we should include the closing curly brace.

![Screenshot 2023-01-05 at 6 24 00 PM](https://user-images.githubusercontent.com/1047763/210917111-8a56c9b4-1e4e-438d-86a5-f360379d9f5c.png)


````suggestion
```js/9-11
````